packages/db-react/tsconfig.json

```diff
@@ -3,6 +3,6 @@
   "compilerOptions": {
     "outDir": "dist"
   },
-  "include": ["src", "test"],
```

Comment:
Are we sure we want to get rid of this? 

I added these here https://github.com/samui-build/samui-wallet/pull/77/files#r2443359062 as my tests were having type issues and `check-types` didn't catch this.

Reply:
Great catch. I have added it back. I don't think we should add folders individually but i'll handle that in the next PR instead.